xml/security_ldap_migrate_test.xml

```diff
@@ -0,0 +1,181 @@
+   <para>
+     This performs a dry-run and does not change anything. The output looks
+     like this:
+   </para>
+   <screen>Examining OpenLDAP Configuration ...
+Completed OpenLDAP Configuration Parsing.
+Examining Ldifs ...
+Completed Ldif Metadata Parsing.
+The following migration steps will be performed:
+ * Schema Skip Unsupported Attribute -> otherMailbox (0.9.2342.19200300.100.1.22)
+ * Schema Skip Unsupported Attribute -> dSAQuality (0.9.2342.19200300.100.1.49)
+ * Schema Skip Unsupported Attribute -> singleLevelQuality (0.9.2342.19200300.100.1.50)
+ * Schema Skip Unsupported Attribute -> subtreeMinimumQuality (0.9.2342.19200300.100.1.51)
+ * Schema Skip Unsupported Attribute -> subtreeMaximumQuality (0.9.2342.19200300.100.1.52)
+ * Schema Create Attribute -> suseDefaultBase (SUSE.YaST.ModuleConfig.Attr:2)
+ * Schema Create Attribute -> suseNextUniqueId (SUSE.YaST.ModuleConfig.Attr:3)
+[...]
+ * Schema Create ObjectClass -> suseDhcpConfiguration (SUSE.YaST.ModuleConfig.OC:10)
+ * Schema Create ObjectClass -> suseMailConfiguration (SUSE.YaST.ModuleConfig.OC:11)
+ * Database Reindex -> dc=example,dc=com
+ * Database Import Ldif -> dc=example,dc=com from example.ldif - excluding entry attributes = [{'structuralobjectclass', 'entrycsn'}]
+No actions taken. To apply migration plan, use '--confirm'
+</screen>
+   <para>
+     The following example performs the migration, and the output looks 
+     different from the dry-run:
+   </para>
+   <screen>&prompt.sudo;openldap_to_ds <replaceable>ldap1</replaceable> /root/slapd.d <replaceable>ldap1-com</replaceable>.ldif --confirm
+Starting Migration ... This may take some time ...
+migration: 1 / 40 complete ...
+migration: 2 / 40 complete ...
+migration: 3 / 40 complete ...
+[...]
+Index task index_all_05252021_120216 completed successfully
+post: 39 / 40 complete ...
+post: 40 / 40 complete ...
+🎉 Migration complete!
+----------------------
+You should now review your instance configuration and data:
+ * [ ] - Create/Migrate Database Access Controls (ACI)
+ * [ ] - Enable and Verify TLS (LDAPS) Operation
+ * [ ] - Schedule Automatic Backups
+ * [ ] - Verify Accounts Can Bind Correctly
+ * [ ] - Review Schema Inconistent ObjectClass -> pilotOrganization (0.9.2342.19200300.100.4.20)
+ * [ ] - Review Database Imported Content is Correct -> dc=ldap1,dc=com
+</screen>
+  
+   <para>
+     When the migration is complete, <command>openldap_to_ds</command>
+     creates a checklists of post-migration tasks that must be completed.
+     It is a best practice to to document all of your post-migration steps,
+     so that you can reproduce them in your post-production procedures.
+     Then test clients and application integrations to the migrated &ds389;
+     instance.
+   </para>
+   <important>
+     <title>Develop a Rollback Plan</title>
+     <para>
+       It is essential to develop a rollback plan with associated back-out
+       criteria to assess the success of your migration.
```

Comment:
So this here means basicly, when you are doing the migration, what are you "go/no-go" criteria. At what point do you say "okay, this is broken, lets undo the changes". And knowing how to do those changes to backout. 

An example could be that you do the migration, and then you have other teams or persons verify that application integrations continue to function IE data feeds, authentication. If these don't work, what of these integrations can be fixed now/later, and what are so critical that we are unable to proceed? 

Does that help?

Reply:
yes!